server/src/com/thoughtworks/go/server/web/IgnoreResolver.java

```diff
@@ -76,6 +77,10 @@ private boolean isGoingToAbout(HttpServletRequest request) {
         return StringUtils.contains(request.getRequestURI(), request.getContextPath() + "/about");
     }
 
+    private boolean isGoingToPlugin(HttpServletRequest request) {
+        return StringUtils.contains(request.getRequestURI(), request.getContextPath() + "/plugins/interact");
```

Comment:
Where did the "/interact" come from? Is ```{requestName}``` above supposed to be "interact"? If so, why use ```{requestName}```?